lib/engine/action/base.rb

```diff
@@ -40,6 +43,7 @@ def to_h
           'entity_type' => type_s(entity),
           'id' => @id,
           'user' => @user,
+          'created_at' => @created_at&.to_i,
```

Comment:
this seems unnecessary, you already have created_at in the db

Reply:
This was also done by @dlpierce. I think you may need it because of hotseat games

Reply:
I didn't think the action timestamp was exposed in the json. If it's available some other way then this is probably not necessary.